content/ngf/overview/gateway-api-compatibility.md

```diff
@@ -124,10 +123,9 @@ See the [static-mode]({{< ref "/ngf/reference/cli-help.md#static-mode">}}) comma
     - `Accepted/False/Invalid`
     - `Accepted/False/UnsupportedValue`: Custom reason for when a value of a field in a Gateway is invalid or not supported.
     - `Accepted/False/GatewayConflict`: Custom reason for when the Gateway is ignored due to a conflicting Gateway.
```

Comment:
It doesn't look like we actually use this condition anymore. so it can be removed.